images/win/scripts/Installers/Install-MysqlCli.ps1

```diff
@@ -11,9 +11,17 @@ $ArgumentList = ("/install", "/quiet", "/norestart")
 Install-Binary -Url $InstallerURI -Name $InstallerName -ArgumentList $ArgumentList
 
 ## Downloading mysql
-$MysqlMajorVersion = (Get-ToolsetContent).mysql.major_version
-$MysqlFullVersion = (Get-ToolsetContent).mysql.full_version
-$MysqlVersionUrl = "https://dev.mysql.com/get/Downloads/MySQL-${MysqlMajorVersion}/mysql-${MysqlFullVersion}-winx64.zip"
+$MysqlVersion = [version](Get-ToolsetContent).mysql.version
+if ($MysqlVersion.Build -lt 0) {
+    $MysqlVersion = New-Object System.Version (
```

Comment:
Why do we need `New-Object System.Version` if Matches.Value is already a full version?

Reply:
Major, Minor and Build fields of System.Version variables are ReadOnly, so unfortunately we can't use something like "$MysqlVersion.Build = ..."

Reply:
I mean `.Matches.Value` is the full version, no need to create System.Version object if we already have it.

`[version]$MysqlVersion = ((Invoke-WebRequest -Uri $Url -UseBasicParsing).Content | Select-String -Pattern "$MysqlMajorMinor\.\d+").Matches.Value`

Reply:
Sorry, got you now. You are right, will correct it. Thanks